<html>
<head>
    <style>
        body {
    background-color: white;
}

h1 {
    color: #002147;
    text-align: center;
}
h2 {
    color: #002147;
    text-align: center;
}
h3 {
    color: #002147;
    text-align: center;
}
ul {
    font-family: verdana;
    font-size: 14px;
}
li {
    font-family: verdana;
    font-size: 14px;
}
p {
    font-family: verdana;
    font-size: 14px;
}
div.scrollmenu {
    background-color: #333;
    overflow: auto;
    white-space: nowrap;
}

div.scrollmenu a {
    display: inline-block;
    color: white;
    text-align: center;
    padding: 14px;
    text-decoration: none;
}

div.scrollmenu a:hover {
    background-color: #777;
}
</style>
<script type="text/javascript">
$(document).ready(function(){
    $('[data-toggle="popover1"]').popover({placement: "auto"});
});
</script>
<style>
  /* Popover Header */
  .popover-title {
      background-color: #B0C4DE;
      color: #FFFFFF;
      height: auto;
      font-size: 28px;
      text-align: center;
      padding: 2px;
  }
  /* Popover Body */
  .popover-content {
      background-color: ;
      color: #000000;
      padding: 10px;
  }
  </style>
</head>
<body>
<div class="scrollmenu">
  <a href="#Main">Main</a>
  <a href="#Project">Project</a>
  <a href="#Results">Results</a>
  <a href="#Log">Log book</a>
  <a href="#Team">Team and Sponsors</a>
</div>
<div class="container-fluid navbarFluid content-main">
<div class="row">
<nav class="col-md-3 bs-docs-sidebar navFont">
        <ul id="sidebar" class="nav nav-stacked" data-spy="affix" data-offset-top="330">
            <li>
                <a href="#Background">Background</a>
                   <ul class="nav nav-stacked">
                       <li><a href="#Gene">Gene expression</a></li>
                       <li><a href="#Riboswitches">Riboswitches</a></li>
                       <li><a href="#Molecular">Molecular Beacons</a></li>
                    </ul>
            </li>
             <li><a href="#Problem">Problem</a></li>
             <li><a href="#Solution">Solution</a></li>
             <li><a href="#Goals">Goals</a>
                   <ul class="nav nav-stacked">
                       <li><a href="#Core">Core</a></li>
                       <li><a href="#Stretch">Stretch</a></li>
                    </ul>
             </li>
             <li><a href="#Future">Future</a></li>
             <li><a href="#Ref">References</a></li>
        </ul>
    </nav>
    </nav>
    </div>

    <div class="col-md-9 content-right" style="background-color: #fff;">
<script>
// Set the date we're counting down to
var countDownDate = new Date("Nov 5, 2017 00:00:00").getTime();

// Update the count down every 1 second
var x = setInterval(function() {

    // Get todays date and time
    var now = new Date().getTime();
    
    // Find the distance between now an the count down date
    var distance = countDownDate - now;
    
    // Time calculations for days, hours, minutes and seconds
    var days = Math.floor(distance / (1000 * 60 * 60 * 24));
    var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
    var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
    var seconds = Math.floor((distance % (1000 * 60)) / 1000);
    
    // Output the result in an element with id="demo"
    document.getElementById("Timer").innerHTML = days + "d " + hours + "h "
    + minutes + "m " + seconds + "s ";
    
    // If the count down is over, write some text 
    if (distance < 0) {
        clearInterval(x);
        document.getElementById("demo").innerHTML = "EXPIRED";
    }
}, 1000);
</script>
<p id="Timer"></p>
<h1 id="Background"> Background (relevance) </h1>
<h3 id="Gene"> Regulated gene expression</h3>
<p>Bioscience industry such as genetic modified organisms and gene therapy are enhanced by the control of gene expression. Control over the expression of the protein-hormone insulin in response to glucose-concentration could, for instance, allow a cell-based treatment for diabetes and selective expression of a toxin only in tumour cells could act as a gene therapy treatment for cancer.</p> 
<p>Previous control of gene expression has largely been achieved through synthetic biology by repurposing natural mechanisms that regulate transcription, such as promoter sequences and protein transcription factors. </p> 
<p>The expression of a gene under the lac promoter, can for instance, be turned on through the addition of the chemical inducer <a data-toggle="popover1" data-trigger="hover" title="IPTG" data-content="Isopropyl β-D-1-thiogalactopyranoside">IPTG</a> that causes unbinding of the repressor protein LacI. </p>

<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/c/cd/LacI_Dimer_Structure_Annotated.png/300px-LacI_Dimer_Structure_Annotated.png" width="20%"/><figcaption>This is a labelled diagram of LacI</figcaption>

<p>Other promoters can couple gene expression to the growth conditions of the cell, its position in the cell cycle or to chemical modifications of DNA-packaging proteins. Through oligomeric proteins and multiple copies of the DNA binding site, often in inverted repeats, protein-protein interactions display cooperative behavior. </p>

<p>Protein-transcription factor systems are not trivial to design as the number of regulated promoter sequences is limited and problems with the dynamic range, size and cell biochemistry requirements limit their use to different species. Building a pathway of many independently regulated genes would prove challenging as they cannot be retargeted to arbitrary DNA sequences.</p> 
<h3 id="Riboswitches">Riboswitches</h3>
<p>Unlike protein-based transcriptional regulation, which is dependent on the complicated tertiary and quaternary folding of the amino acids of a protein sequence and the interaction of the protein with DNA, control of the level of mRNA structure can be predicted and designed through simpler nucleotide interactions. </p>
<p>In general stable mRNA structures, such as RNA hairpins, that obstruct the binding of the ribosome to the ribosome-binding site (rbs) lower the rate of translation of the mRNA producing less of the protein product. (Salis et al. 1) </p>
<p>Riboswitches (RNA switches) have great potential for the design of synthetic genetic circuits. Riboswitches are sequences of RNA that binds small molecules (RNA aptamers) that are located within a protein-encoded mRNA sequence. The binding of the small molecule changes the structure of the riboswitch exposing or hiding the ribosome binding site and thereby controlling gene expression. Other riboswitches operate through RNA self-cleavage (e.g. GlmS ribozyme) or by attenuation of transcription.</p>
<p>Natural riboswitches are present in bacteria allowing regulated gene expression in response to amino acids and metal ions. </p>

Riboswitch exp. eg. Gif of Gly ribsowitch


</p>Using RNA aptamers generated by <i>in vitro</i> evolution/SELEX(popover) synthetic riboswitches have been produced that respond to a variety of chemicals. (2-Salis diverse chemical riboswitches).</p>
<p>Yin et al.(3) have made riboswitches at the 5’ end of mRNA that are responsive to the presence of a Trigger strand of RNA that anneals to a single strand region to open a hairpin (Toehold mechanism). Opening this hairpin reveals an rbs located in the hairpin loop. By investigating different sequences they managed to achieve ON/OFF ratios greater than 600. As the sequence of the trigger was not greatly restricted multiple versions can be created and they demonstrated independent regulation of four different fluorescent protein reporters. </p> 
<p>The AUG start codon in these Toehold switches is located in the stem of the hairpin necessitating a translated linker before the protein of interest.</p>
<h3 id="Molecular"> Molecular Beacons</h3>
<p>Molecular beacons are DNA or RNA molecules that form hairpin structures typically with the ends labeled with fluorophore-quencher pairs. The binding of a complementary sequence to the loop region is able to unwind the hairpin to separate the fluorophore-quencher pair leading to a detectable fluorescence output. Molecular beacons can be used for detecting specific DNA or RNA sequences in a sample for instance the detection of viral RNA. </p>
<p>Plaxco et al.(4) created molecular beacons that had the property of cooperativity with respect to their trigger i.e. instead of a broad dose-response relationship there was switch-like behaviour. </p>
Diagram of coop. molecular beacon in three states e.g fig3
<p>Cooperativity in these systems arises because the first trigger binding event is of low affinity, in this case because it requires the unfavourable unwinding of the stem of the hairpin, whilst the second binding event is of greater affinity, as the hairpin stem in already unwound and the complementary bases are exposed.</p>
<p>The degree of cooperativity, quantified by the hill coefficient n, rapidly approaches its maximum (equal to the number of binding sites) as the affinity of the second binding event trends to infinitely more favorable than the first.</p>
<p>Plaxco also generated cooperative molecular receptors, using a different design consisting of two tandem copies of one half of an aptamer, a long unstructured linker, then two copies of the second half of the aptamer.(5)</p>
Diagram e.g. fig2 of generalizable
<p>Cooperativity arises in this design as the first binding event must overcome the unfavourable free energy of folding of the unstructured linker to bring together both halves of the aptamer. The second binding event is more favourable as the two halves of the aptamer have already been brought close together. Binding both ligands brings together the ends of the receptor that are labeled with a fluorophore-quencher pair.</p> 
<p>Consistent with this interpretation as the length of the linker increases the more cooperative the receptor becomes.</p>
<p>Plaxco demonstrated these receptors using split aptamers for mercury, doxorubicin and cocaine. Ricci et al. (6) created a molecular beacon of a similar design. This beacon detected 9bp DNA sequence through two repeats of a clamp-like binding site where one half of the clamp formed Watson-Crick base pairs to the ligand and one half Hoogsteen base pairs to the same ligand. The two half were separated by a similar disordered loop. </p>
Diagram of their molecular beacon.
<h1 id="Problem"> The Problem (relevance) </h1>
<ul>
<li> Protein-promoter regulation of gene products is limited by low dynamic range, species specificity and size.</li>
<li>Protein-promoter interactions cannot easily be redesigned.</li>
<li>Riboswitches are easier to implement than protein-promoter systems and can be triggers by short pieces of RNA. However riboswitches responsive to oligonucleotide triggers lack the cooperativity of protein-promoter interactions and are often context-dependent and can misfold due to surrounding sequences. </li>
</ul>
<h1 id="Solution"> Our solution (merit) </h1>
<p>We will design a set of riboswitches that can be controlled, in a cooperative fashion, by the concentration by a oligonucleotide Trigger strand through the presence of multiple binging sites.</p>

<h1 id="Goals"> Project Goals (Specification and Feasibility)</h1>
<h3 id="Core">Core Goals</h3>
<ul>
<li>Design a two-site activatory riboswitch that, similar to the Plaxco molecular beacons operates through unwinding of an hairpin, is able to respond cooperatively to the concentration of a trigger strand allowing the ribosome to access a previously hidden rbs.</li>
<ul>
<li>Demonstrate this cooperative behaviour <i>in vitro</i> using fluorophore-quencher pairs in comparison to non-cooperative, single-site controls using the same trigger sequence.</li>
<li>Demonstrate cooperative control of gene expression using the expression of a fluorescent protein reporter in <i>E. coli</i> cells whilst measuring the concentration of our trigger RNA via fluorescence of a broccoli RNA tag.</li>
<li>Demonstrate that we can tune our switches for different applications via changing the Trigger strand Kd.</li>
<ul>
<li>Investigate increasing the stem length of the switch from 8 to 12 to 16. This should increase the Kd of the first Trigger binding event and increase the Hill coefficient.</li>
</ul>
</ul>
<li>Design a two-site repressive riboswitch that, similar to the second type of Plaxco cooperative molecular beacons that operates the bringing together of split binding sites, is able to respond cooperatively to the concentration of a trigger strand hiding a previously accessible rbs</li>
<ul>
<li>Similarly demonstrate this behaviour <i>in vitro</i> and <i>in vivo</i>.</li>
<li>Investigate cooperativity of a repressive design by varying loop length and number of binding sites.</li>
<ul>
<li>Cooperativity, as measured by the hill coefficient, should increase as loop length and number of binding sites increases.</li>
</ul>
</ul>
</ul>
<h3 id="Stretch">Stretch Goals</h3>
<ul>
<li>Compare homotropic (same trigger) and heterotropic (two different triggers) cooperativity for both activatory designs. </li>
<ul>
<li>Investigate heterotropic cooperativity <i>in vitro</i> using fluorophore-quencher pairs by adding different concentrations of the two trigger strands.</li>
<li>Investigate cooperativity <i>in vivo</i> by expressing both triggers as a single construct separated by a TPP-sensitive ribozyme. Addition of TPP should lead to the separation of the triggers and their independent action.</li>
</ul>
<li>Investigate other riboswitch designs including a Triple-trigger design, a homotropic asymmetric design and a cooperative Toehold switch.</li>
</ul>
</ul>
<h1 id="Future">Future Work</h1>
<ul>
<li>Perfect the switches to increase n. </li>
<li>Implement attenuator by Folliard et al.(7) to allow the switches to be applied to any protein. Contextual sensitivity of riboswitch</li>
<li>Investigate other variable affecting switch activation e.g. trigger length, rbs strength and rbs position.</li>
<li>Incorporate multiple switches into one system to show independent regulation.</li>
<li>Exchange the Broccoli aptamer sequence tag for another RNA aptamer e.g. an aptamer for the output of the system creating a feedback loop.</li>
</ul>
<h1 id="Ref">References</h1>
<p>(1) Salis: hairpins in N terminal regions</p>
<p>(2) Salis: diverse chemical riboswitches</p>
<p>(3) Yin: toehold switches</p>
<p>(4) Plaxco: pop-shift</p>
<p>(5)PLAXCO: intrinsic disorder</p>
<p>(6)Plaxco: haemoglobin-like</p>
<p>(7)Folliard et. al. </p>
</body> </html>
